lesson07.blog-api-server/server.js

```diff
@@ -28,10 +175,16 @@ http.createServer(function (req, res) {
             }
         } else {
             res.writeHead(200, {'Content-Type': cType});
+            if (cType === 'application/json') {
+                // data manipulation
+                const dataToView = getDataFromRoutes(req.url, JSON.parse(data));
+                data = JSON.stringify(dataToView, null, 2);
```

Comment:
In case of `/comments.json` your will read the comments file, in addition to the pre-fetched data (line 15)
so route `/comments` will result in reading the files 2 times.